variant-remapping-automation/genome_target_tracker.py

```diff
@@ -0,0 +1,96 @@
+from argparse import ArgumentParser
+
+import requests
+from ebi_eva_common_pyutils.logger import logging_config
+from ebi_eva_common_pyutils.metadata_utils import get_metadata_connection_handle
+from ebi_eva_common_pyutils.pg_utils import get_all_results_for_query
+from ebi_eva_common_pyutils.taxonomy.taxonomy import get_scientific_name_from_ensembl
+
+logger = logging_config.get_logger(__name__)
+logging_config.add_stdout_handler()
+
+remapping_genome_target_table = 'evapro.remapping_genome_target_tracker'
```

Comment:
Do we want to tie the name of this table to the remapping process?  If we intend to use it more generally as a "current supported assembly" kind of thing I suggest making the name more general as well.

Reply:
renamed to evapro.supported_assembly_tracker